website/blog/2023-06-05-ferretdb-new-release-v-1-3-0.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: ferretdb-new-release-v-1-3-0
+title: FerretDB v1.3.0. Release
+authors: [alex]
+description: >
+  We’ve just announced the release of a new version of FerretDB v1.3.0, which now includes new feature additions, such as the `logout` command and positional operator in projection.
+image: /img/blog/ferretdb-v1.3.0.jpg
+tags: [release]
+---
+
+![FerretDB v.1.3.0](/img/blog/ferretdb-v1.3.0.jpg)
+
+We've just announced the release of FerretDB v1.3.0, which includes new feature additions, such as the `logout` command and positional operator in projection.
+
+<!--truncate-->
+
+Since our [GA release](https://blog.ferretdb.io/ferretdb-1-0-ga-opensource-mongodb-alternative/), we've worked on improving all aspects of [FerretDB](https://www.ferretdb.io/) with the help of our active and growing community; you've all been truly amazing!
+
+We've seen a high demand for FerretDB and its application as an open-source replacement for MongoDB workloads and applications.
+This is exciting news, and we look forward to enabling more developers build their applications with FerretDB, starting with the latest release.
+
+Want to know what's new in FerretDB v.1.3.0?
+Let's find out.
+
+## New features
+
+In this release, we've added the `logout` command.
+Even though it's a deprecated in MongoDB, it's necessary for our compatibility tests.
```

Comment:
```suggestion
Even though it's deprecated in MongoDB, it's necessary in some circumstances.
```

Otherwise, we are very excited about implementing a deprecated command just for compatibility tests